src/PrestaShopBundle/Resources/views/Admin/Product/ProductPage/disabled_form_alert.html.twig

```diff
@@ -25,6 +25,9 @@
 {% extends '@PrestaShop/Admin/layout.html.twig' %}
 
 {% block content %}
+  <div class="header-toolbar d-print-none">
+    {{ multistoreHeader()|raw }}
```

Comment:
You should use twig options to prevent unneeded escape:
https://twig.symfony.com/doc/3.x/advanced.html#automatic-escaping